R/spread_with_duplicates.R

```diff
@@ -0,0 +1,76 @@
+#' Spread a key-value pair across multiple columns in presence of duplicates
```

Comment:
duplicates of *what* exactly? This is not so easy to understand in the beginning. It basically means that for these two rows all the non-key and no-value columns have the same values for a specific key...

Reply:
You are right. I changed it using `spread()` documentation as template. See a9cba11